packages/devtools_app/lib/src/performance/flutter_frames_chart.dart

```diff
@@ -315,13 +319,23 @@ class FlutterFramesChartItem extends StatelessWidget {
             color: defaultSelectionColor,
             height: selectedIndicatorHeight,
           ),
+        if (frame.hasShaderTime)
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: const [
+              Icon(Icons.warning_amber_rounded),
+            ],
+          ),
       ],
     );
   }
 
   String _tooltipText(FlutterFrame frame) {
-    return 'UI: ${msText(frame.uiEventFlow.time.duration)}\n'
-        'Raster: ${msText(frame.rasterEventFlow.time.duration)}';
+    return [
+      'UI: ${msText(frame.uiEventFlow.time.duration)}',
+      'Raster: ${msText(frame.rasterEventFlow.time.duration)}',
+      if (frame.hasShaderTime) 'Skia.Shaders: ${msText(frame.shaderDuration)}',
```

Comment:
Add a link to the tooltip and a message explaining that shader compilation jank can usually be eliminated with caching.

Reply:
AFAIK flutter doesn't support rich tooltips with links, so we'll likely have to use the hovercard impl we use in the debugger for this. I'll look into that.

Reply:
added a TODO to do this in a follow up PR

Reply:
One other idea is to use the sort of overlay we use to warn about profile vs debug mode to warn about this.
The overlay could summarize the total impact of shader jank for the CPU profile. For example: 
"7 frames janked due to shader compilation with a total of ???ms spent on shader compilation.
To precomple shaders, see the instructions on this doc. That also has the benefit that users do not have to hover to see the message. We could also dispatch this as an error so it is shown to IDE users using the same Flutter.Error used for errors dispatched from Flutter. Fine to do that as a follow up PR.